apps/examples/src/examples/export-canvas-as-image/README.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Export canvas as image
+component: ./ExportCanvasImageExample.tsx
+category: basic
+priority: 1
```

Comment:
Not sure if I'd put it on the top. At least behind the basic example.

Reply:
Ofc!

Reply:
Actually I've moved it over to data/assets as well